app/assets/javascripts/discourse/app/components/bookmark-menu.gjs

```diff
@@ -59,18 +58,67 @@ export default class BookmarkMenu extends Component {
       return I18n.t("bookmarks.not_bookmarked");
     } else {
       if (this.existingBookmark.reminderAt) {
-        return I18n.t("bookmarks.created_with_reminder", {
+        return I18n.t("bookmarks.created_with_reminder_generic", {
           date: this.existingBookmark.formattedReminder(this.timezone),
           name: this.existingBookmark.name || "",
         });
       } else {
-        return I18n.t("bookmarks.created", {
+        return I18n.t("bookmarks.created_generic", {
           name: this.existingBookmark.name || "",
         });
       }
     }
   }
 
+  get buttonClasses() {
```

Comment:
I just felt this was getting too complex for `concatClass`

Reply:
I don't think so, but either way it's fine :D